platform/root/index.ts

```diff
@@ -36,9 +36,16 @@ export class Root {
   }
 
   async start() {
-    const loggingConfig$ = this.configService.atPath('logging', LoggerConfig);
-
-    this.loggerService.upgrade(loggingConfig$);
+    try {
+      const loggingConfig$ = this.configService.atPath('logging', LoggerConfig);
+      this.loggerService.upgrade(loggingConfig$);
+    } catch (e) {
+      // This specifically console.logs because we were not able to configure
+      // the logger.
+      console.log('Configuring logger failed with the following errors:');
```

Comment:
__question:__ why not do it in a single `console.log` call?
__question:__ does it make sense to call `console.error` instead (to write to `stderr`)?